test/unit/chruby_reset_test.sh

```diff
@@ -15,10 +15,13 @@ function test_chruby_reset_hash_table()
 		assertEquals "did not clear the path table" \
 			     "" "$(hash)"
 	elif [[ -n "$BASH_VERSION" ]]; then
+		export LANG=en_US.UTF-8
```

Comment:
We should probably do this for `zsh` as well, yes?

Reply:
The Zsh test above only compares it with empty string, which doesn’t sound like language-specific.